### ISSUE TITLE
(c) year in "about" dialogue is still at 2020

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbase/vorta#1838]
Just a minor thing, title says it all.

- OS: macOS
- Vorta version: 0.9.1-beta2
- Installed from: dmg

**Comment by real-yfprojects** [comment]
Copyright doesn't have to be updated.

**Comment by Akarn-26** [comment]
@ThomasWaldmann hi. How can I reproduce this issue ?

**Comment by ThomasWaldmann** [comment]
If somebody fixes that, also fix the grammar issue in the about dialogue:
- it says "click here **to** view the docs." (correct)
- below that it says "click here **for** view git repo." (incorrect)

**Comment by pavankalyan224847** [comment]
iam fixing this issue should i update the copyrights(C) from 2018 to 2023 or 2024 ?

**Comment by m3nu** [comment]
2024

**Comment by pavankalyan224847** [comment]
yes updated it

**Comment by real-yfprojects** [comment]
> The year of a copyright doesn't have to be updated afaik.

https://nexb.com/do-you-really-need-to-update-the-copyright-each-new-year/

**Comment by m3nu** [comment]
> > The year of a copyright doesn't have to be updated afaik.
> 
> https://nexb.com/do-you-really-need-to-update-the-copyright-each-new-year/

I agree, but still looks a bit more maintained. Also a good first issue for someone who wants to look at the code the first time.

**Comment by shivansh02** [comment]
Can this be assigned to me? No one seems to be working on this at the moment.

**Comment by m3nu** [comment]
All yours.

There is also a related PR #1909 open that has gone a bit stale. And seems too big for the planned change. Maybe ping the author if he didn't manage to set up Qt Design.
